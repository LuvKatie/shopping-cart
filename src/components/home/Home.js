import React from "react";
import "../../styles/home.css";

const Home = () => {
  return <main id="home-page" className=""></main>;
};

export default Home;
