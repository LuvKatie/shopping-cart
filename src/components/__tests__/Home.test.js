import React from "react";
import {render, screen} from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import Home from "../home/Home";

describe("Home component", () => {
    it("renders Home component", () => {
        const { container } = render(<Home />);
        expect(container).toMatchSnapshot();
    });
})