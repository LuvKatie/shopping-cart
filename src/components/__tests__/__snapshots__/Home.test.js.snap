// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders Home component 1`] = `
<div>
  <div />
</div>
`;
